core/server/common/src/main/java/alluxio/cli/validation/AbstractValidationTask.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.validation;
+
+import alluxio.exception.status.InvalidArgumentException;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * Abstract class for validation environment.
+ */
+public abstract class AbstractValidationTask implements ValidationTask {
+  protected static final Option HADOOP_CONF_DIR_OPTION =
```

Comment:
Let's not put Hadoop specific option in `AbstractValidationTask`. Instead of letting each task parse command line own its own, we can just expose an interface method `Option[] getOptions()`. The validateEnv class can call each validationTask to get their specific list of options, and it can build the single `Options` object which can be used to parse the command line and print unified help message which will contain explanation of all options.

Reply:
Currently, `AbstractValidationTask` already exposes a `getOptions` method. Furthermore, it also implements a `parseArgsAndOptions` method. This is similar to Alluxio CLI.

Reply:
Done.